### ISSUE TITLE
Accessibility: click handler for brand even when there's no URL configured

### ISSUE THREAD

**Issue by MikeTschudi** [Esri/global-nav#301]
header-brand.js has
```javascript
// On Click
$target.addEventListener('click', (event) => {
  $dispatch($target, 'header:click:brand', {event});
});
```
Can this be made optional if there's no URL configured? As is, we get an accessibility error "Clickable elements must be focusable and should have interactive semantics."
